### PR TITLE
(#138) Updated Sprites Test Code 

### LINKED ISSUE #138: Update sprites test code
### Issue description
*Description of the issue*
per https://github.com/NCIOCPL/cgov-digital-platform/issues/2782
Special report test should get rid of the fingerprinting part of url
replace code with 
```
 const baseURL = Cypress.config('baseUrl');
        cy.document().then((document) => {
            const element = document.querySelector("div[class='ap-file-block pdf'] a");
            const style = window.getComputedStyle(element, '::before');
            const image = style.getPropertyValue('background-image');
            const expectedURL= `url("${baseURL}${iconUrl}")`;
            const newImage = image.replace(/\?v=\w+/,''); // here is the change
            expect( expectedURL).equal(newImage); //and here
           const text=element.innerText;
           expect(text).equal(iconText);
        });
```
> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  
2. 
3. 

### What's the expected result?
-

### What's the actual result?
-

### Additional details / screenshot
- Screenshot

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by alinai**
Updated contentBlockButton.js and contentBlock.js to fix the error described on https://github.com/NCIOCPL/cgov-digital-platform/issues/2782

Closes #138

### CODE REVIEW

**Comment by alinai** [comment]
Test Result:

image
image

**Comment by alinai** [comment]
@KateMashkinaNIH , please review. Thanks!

**Comment by alinai** [comment]
image

File: cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js [hunk 1]
```diff
@@ -5,28 +5,28 @@ import { Given, Then } from "cypress-cucumber-preprocessor/steps";
...
-           expect(text).equal(iconText);
-        });
+And('a PDF icon located at {string} should be displayed with a text {string}', (iconUrl, iconText) => {
+    const baseURL = Cypress.config('baseUrl');
+    cy.document().then((document) => {
+        const element = document.querySelector("div[class='ap-file-block pdf'] a");
+        const style = window.getComputedStyle(element, '::before');
+        const image = style.getPropertyValue('background-image');
+        const expectedURL = `url("${baseURL}${iconUrl}")`;
+        const newImage = image.replace(/\?v=\w+/, ''); // here is the change
```
Comment (kate-mashkina) [inline cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js:17]: please remove the comments ))) they were meant to explain the ticket and should not be reflected in the code

File: cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js [hunk 2]
```diff
@@ -5,28 +5,28 @@ import { Given, Then } from "cypress-cucumber-preprocessor/steps";
...
+        expect(expectedURL).equal(newImage); //and here
+        const text = element.innerText;
+        expect(text).equal(iconText);
     });
+});
```
Comment (kate-mashkina) [inline cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js:23]: also nothing below this line should be showing any changes.. Can you revert formatting ?

File: cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js [hunk 3]
```diff
@@ -31,6 +31,6 @@ Then('the PDF download link is not displayed', () => {
 });
 
 When('user clicks on top {string} button', (hrefAttr) => {
-    cy.get("div[class='ap-file-block pdf']").find('a').trigger('click', {followRedirect:false});
+    cy.get("div[class='ap-file-block pdf']").find('a').trigger('click', { followRedirect: false });
```
Comment (kate-mashkina) [inline cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js:34]: just revert formatted change here - it's confusing to see a difference

File: cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js [hunk 4]
```diff
@@ -5,19 +5,19 @@ import { Given, Then } from "cypress-cucumber-preprocessor/steps";
...
         const element = document.querySelector("div[class='ap-file-block pdf'] a");
         const style = window.getComputedStyle(element, '::before');
         const image = style.getPropertyValue('background-image');
-        const expectedURL= `url("${baseURL}${iconUrl}")`;
-        expect( expectedURL).equal(image);
-       const text=element.innerText;
-       expect(text).equal(iconText);
+        const expectedURL = `url("${baseURL}${iconUrl}")`;
+        const newImage = image.replace(/\?v=\w+/, ''); // here is the change
+        expect(expectedURL).equal(newImage); //and here
```
Comment (kate-mashkina) [inline cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js:18]: same with comments - please remove them

**Review by kate-mashkina — CHANGES_REQUESTED** [review]

File: cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js (hunk 1 above)
Comment (alinai) [reply cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js]: Done

File: cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js (hunk 3 above)
Comment (alinai) [reply cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js]: Done

File: cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js (hunk 4 above)
Comment (alinai) [reply cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js]: Done

File: cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js (hunk 2 above)
Comment (alinai) [reply cypress/integration/AnnualPlan/ContentBlockAPButton/contentBlockButton.js:24]: Done

**Comment by alinai** [comment]
@KateMashkinaNIH , updated per your review. Please review again.

**Review by KateMashkinaNIH — APPROVED** [review]

**Pull request merged**